ha/util/consul_kv_store.py

```diff
@@ -120,9 +143,20 @@ def set(self, key: str, val: str=None):
             str: Return value.
         """
         self._verify_data(key)
+
         if self.key_exists(key):
             raise Exception(f"Key {key} already exists in kv store.")
-        self._consul.kv.put(self._prepare_key(key), val)
+
+        k = self._prepare_key(key)
+
```

Comment:
Too airy.

Reply:
deleted empty lines. if you mean 'k' will change for the whole file with some meaningful name let me know.
I have just tried to keep consistent naming.

Reply:
will fix in: https://jts.seagate.com/browse/CORTX-31319